cmake/GenerateRequirementsTable.py

```diff
@@ -19,9 +20,15 @@
 parser.add_argument('--ext', action='append', type=str,
                     help='The file extensions to check in for requirements', default=[])
 parser.add_argument('--output', type=str,
-                    help='Name of the output file', default='requirements.html')
+                    help='Base name of the output file', default='requirements')
 parser.add_argument('--skip-no-require', dest='skip_no_require', action='store_true',
                     help='If added, files with no requirements present will not be added to the HTML file')
```

Comment:
I really think we ought to have a couple tests on this utility because of what we're using it for.  Also, seeing an example of what this file that it is trying to process looks like will help me in setting something similar up for CTF.

Reply:
I will add some tests

Reply:
Doesn't look like a test was ever added for this?

Reply:
Tests are now added.